README.md

```diff
@@ -476,6 +476,7 @@ You can find other awesome lists related to Xamarin below:
 - [Xamarin.Forms.VideoPlayer](https://github.com/adamfisher/Xamarin.Forms.VideoPlayer): A Xamarin Forms control to render the native video player on every platform.
 - [Xam.Plugin.SimpleAppIntro](https://github.com/galadril/Xam.Plugin.SimpleAppIntro): Just a nice and simple AppIntro for your Xamarin Forms project.
 - [Xam.Plugin.SimpleColorPicker](https://github.com/galadril/Xam.Plugin.SimpleColorPicker):  Just a nice and simple ColorPicker for your Xamarin Forms project.
+- [Xam.Plugin.SimpleBottomDrawer](https://github.com/galadril/Xam.Plugin.SimpleCBottomDrawer):  Just a nice and simple BottomDrawer for your Xamarin Forms project.
```

Comment:
the newly added url is unreachable, the C shouldn't be in there in my opinion :)
https://github.com/galadril/Xam.Plugin.SimpleBottomDrawer instead of https://github.com/galadril/Xam.Plugin.SimpleCBottomDrawer

Reply:
woops.. indeed

Reply:
Would be fixed now